xbmc/cores/VideoPlayer/DVDCodecs/Video/DVDVideoCodecFFmpeg.cpp

```diff
@@ -339,6 +339,10 @@ bool CDVDVideoCodecFFmpeg::Open(CDVDStreamInfo &hints, CDVDCodecOptions &options
     return false;
 
   UpdateName();
+
+  if (hints.pPacket)
+    Decode(hints.pPacket->pData, hints.pPacket->iSize, hints.pPacket->dts, hints.pPacket->pts);
```

Comment:
this is definitely the wrong place to do this. if required (please elaborate) it needs to be done by player